libraries/chain/son_evaluator.cpp

```diff
@@ -188,7 +188,7 @@ object_id_type son_report_down_evaluator::do_apply(const son_report_down_operati
             });
 
             db().modify(*itr, [&op](son_object &so) {
-                so.status = son_status::in_maintenance;
+                so.status = son_status::request_maintenance;
```

Comment:
missing heartbeats without requesting maintenance will result in moving the SON to in_maintenance because the SON is somehow down and didn't send heartbeats.

Reply:
ok,  I'll fix it